### DISCUSSION TITLE
Getting users avatar from Discord when using Discord log-in

### DISCUSSION THREAD

**Discussion by schwarzsky** [supabase/supabase#3334] [Questions] [answered]
I used quickstart NextJS template and used `User Management Starter` for SQL. Enabled log-in with Discord.

`handleLogin()`
```
const handleLogin = async () => {
    await supabase.auth.signIn({provider: 'discord'})
 }
```

When i login with Discord, `avatar_url` not setted.
https://soo.gg/u/qLO0Msa9.png

Anyone have idea?

**Comment by silentworks** [comment]
This has been discussed before I think there was an issue on the gotrue-js library repo. But I think one of the issue is that not all OAuth providers send back an avatar.

**Reply by schwarzsky** [reply]
Discord provides avatar on identify object, so this is default object if you need to auth with Discord. 

I found a issue about that, anyone want to access can use `supabase.auth.user().user.user_metadata?.avatar_url`.

Thanks for reply.

**Comment by schwarzsky** [comment, marked as answer]
I found a issue about that, anyone want to access can use `supabase.auth.user().user.user_metadata?.avatar_url`.
